src/NHSD.BuyingCatalogue.Ordering.Common/Results/Result.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using NHSD.BuyingCatalogue.Ordering.Common.Models;
+
+namespace NHSD.BuyingCatalogue.Ordering.Common.Results
+{

```

Comment:
This class was adding as part of my changeset and lives in the domain layer, if you want to move it to common thats fine, but remove the one in the domain layer

Reply:
I think it just needs deleting , i moved to the domain version once i picked them up from your code. Will delete them